src/test_lib_json/main.cpp

```diff
@@ -2045,7 +2045,19 @@ JSONTEST_FIXTURE(CharReaderTest, parseWithStackLimit) {
     delete reader;
   }
 }
-
+JSONTEST_FIXTURE(CharReaderTest, issue774) {
+  Json::CharReaderBuilder b;
+  Json::CharReader* reader(b.newCharReader());
+  Json::String errs;
+  Json::Value root;
+  char const doc[] = "-i";
+  bool ok = reader->parse(doc, doc + std::strlen(doc), &root, &errs);
+  JSONTEST_ASSERT(!ok);
```

Comment:
I think the CI build failed just because the `ok` is `true`, but actually it shouldn't. So I submit a PR(#941).